firefox_media_tests/playback/test_video_playback.py

```diff
@@ -24,20 +21,25 @@ class TestVideoPlayback(MediaTestCase):
     def test_playback_starts(self):
         with self.marionette.using_context('content'):
             for url in self.video_urls:
-                self.logger.info(url)
-                video = VideoPuppeteer(self.marionette, url)
-
                 try:
-                    verbose_until(Wait(video, timeout=30), video,
-                                  playback_started)
+                    video = VideoPuppeteer(self.marionette, url, timeout=60)
+                    # Second playback_started check in case video._start_time
+                    # is not 0
+                    self.check_playback_starts(video)
+                    video.pause()
+                    src = video.video_src
+                    if not src.startswith('mediasource'):
+                        self.marionette.log('video is not '
+                                            'mediasource: %s' % src,
+                                            level='WARNING')
                 except TimeoutException as e:
                     raise self.failureException(e)
 
-                video.pause()
-                src = video.video_src
-                if not src.startswith('mediasource'):
-                    self.marionette.log('video is not mediasource: %s' % src,
-                                        level='WARNING')
-
     def test_video_playback_partial(self):
-        self.run_playback(set_duration=60)
+        """ First 60 seconds of video play well. """
+        with self.marionette.using_context('content'):
+            for url in self.video_urls:
+                video = VideoPuppeteer(self.marionette, url,
+                                       stall_wait_time=10,
+                                       set_duration=60)
```

Comment:
Should probably put timeout=60 in this call as well, or change VideoPuppeteer's default.